client/src/components/web-extension/NFTs/NFTDetails/components/NFTConfirmTransfer.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { DEPLOY_TYPES } from '@cd/constants/deployTypes';
+import { getTokenStandardName } from '@cd/helpers/nft';
+import { MiddleTruncatedText } from '@cd/components/Common/MiddleTruncatedText';
+
+import './NFTConfirmTransfer.scss';
+
+export const NFTConfirmTransfer = ({nftDetails}) => {
+    const listItems = [
+        {
+          title: 'Name',
+          value: nftDetails.name,
+        },
+        {
+          title: 'Token ID',
+          value: `#${nftDetails.tokenId}`,
+        },
+        {
+          title: 'Receiving Address',
+          value: <MiddleTruncatedText end={8}>{nftDetails.receivingAddress}</MiddleTruncatedText>,
+        },
+        {
+          title: 'Contract Package Hash',
+          value:  <MiddleTruncatedText end={8}>{nftDetails.contractAddress}</MiddleTruncatedText>,
+        },
+        {
+          title: 'Token Standard',
+          value: getTokenStandardName(nftDetails.tokenStandardId),
+        },
+        {
+          title: 'Transfer Type',
+          value: nftDetails.isUsingSessionCode ? DEPLOY_TYPES.WASM : DEPLOY_TYPES.CONTRACT_CALL,
+        },
+        {
+          title: 'Network Fee',
+          value: `${nftDetails.fee} CSPR`,
```

Comment:
Just wondering is there anycase `nftDetails` would become `undefined` ?

Reply:
I think no for this case
<img width="450" alt="image" src="https://github.com/CasperDash/casperdash-extension/assets/20489407/d71cbcab-e897-40e0-9edf-c7c77c9d67de">